.aicoe-ci.yaml

```diff
@@ -0,0 +1,5 @@
+check:
+  - thoth-tox
+  - thoth-precommit
+release:
+  - upload-pypi-sesheta
```

Comment:
Not sure if this will work - we use a single file micropipenv.py where the version is kept, whereas kebechet will look for other places.

Reply:
This is to enable the pipeline to make the release, which basically just building the module and release with help of twine.
the pipeline is trigger on tag release, so it directly doesn't look for version. only utilize `setup.py` .

this was just added if we want to make the release automated by the pipeline. 
maintainer would have to just create a tag with relevant changes.